common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -29,11 +29,12 @@
     </li>
 }
 
-<div class="main-menu-container" id="main-menu">
+<div href="#" class="main-menu-container__overlay" aria-hidden="true"></div>
```

Comment:
Sharp eyes @regiskuckaertz ! This was an `a` tag, but with the new and improved implementation it doesn't make sense for it to be that anymore. I think @SiAdcock is going to investigate making it either a label (so will uncheck the checkbox) or it will only be a functional way of closing the menu if Javascript is involved.